freemocap/core_processes/capture_volume_calibration/anipose_camera_calibration/anipose_camera_calibrator.py

```diff
@@ -289,4 +291,23 @@ def adjust_world_reference_frame_to_charuco(self,
                 new_rmat = rmat_world_to_cam_i @ rmat_charuco_to_world #composes w' -> w (rotation matrix) with w -> c (rmat) to get w' -> c
                 new_rvec, _ = cv2.Rodrigues(new_rmat)
                 rvecs_new[i] = new_rvec.flatten()
-            return Extrinsics(rvecs_new, tvecs_new)
\ No newline at end of file
+            return Extrinsics(rvecs_new, tvecs_new)
+    
+    def get_real_world_matrices(self,
```

Comment:
I think you can adjust this method to put each camera's world 6DoF into its existing entry, just tweak the calculation and `metadata` update (`cam_group.metadata["camera_positions"] = tvecs_world`) to key into the `cam_group` object by camera name